conf.py

```diff
@@ -59,7 +59,7 @@
 master_doc = 'index'
 
 # General information about the project.
-project = 'AT-TPC Analysis'
+project = 'AT-TPC_Analysis'
```

Comment:
Great, this fix seems to work. Another weird thing occurs with the autonumbering that is used in the pdf format. This casues sections to be double numbered due to the step numbers that I actually write in the .rst files. These are necessary for the HTML rendering though so I guess we should ignore it.